tests/installation/bootloader_s390.pm

```diff
@@ -251,6 +248,7 @@ sub format_dasd {
     $r = script_run("dasd_configure $dasd_path 1");
     die "DASD in undefined state" unless (defined($r) && ($r == 0 || $r == 8));
 
+    assert_script_run('dasd_reload');
```

Comment:
hmm that's only in the encrypted case now, right?

Reply:
It's for normal format, just tried reload. The dasd gets into faulty state and can;t be reloaded or deactivated.
https://openqa.suse.de/tests/3943946#step/bootloader_s390/59